lmdeploy/serve/openai/api_server.py

```diff
@@ -360,6 +360,7 @@ async def chat_completions_v1(request: ChatCompletionRequest,
 
     gen_config = GenerationConfig(
         max_new_tokens=request.max_tokens,
+        do_sample=True,
```

Comment:
shall we make it a request option?

Reply:
I think it's unnecessary since users can still use `top_k=1` or `temperature=0.` to invoke greedy search